### PR TITLE
Add tests for geo functions

### PR DESCRIPTION
**Pull request by atombender**
This is a starting point and not an exhaustive set of tests. For `geo::contains()` and `geo::intersect()`, I got some real map boundaries that turned out to be rather large, even after running through a simplifier, and it'd be nice to reduce them a bit.

### CODE REVIEW

**Comment by j33ty** [comment]
Can we use an external geo dataset like movies dataset and then not care to minimize the size so that we can test on non-simplified inputs.

File: test/function/geo/contains.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+tests:
+- name: Point in multipolygon
+  query: |
+    * { _id, "contains": geo::contains(geo(boundary), geo(~point~)) }
```
Comment (judofyr) [inline test/function/geo/contains.yml:14]: Add some tests which uses `geo::contains(boundary, ~point~)` as well?

**Comment by atombender** [comment]
I'm going to add some more tests for `geo::contains()` and `geo::intersects()`.

**Review by judofyr — COMMENTED** [review]
Looks good.

Some additional tests which don't use the geo-constructor would be nice as well!

**Comment by atombender** [comment]
I've added additional tests that _also_ use `geo()`. There's lots of repeated data here, but I don't see a way to avoid it without introducing some kind of macro/variable system.

**Review by j33ty — APPROVED** [review]
